src/ring_theory/ideal_operations.lean

```diff
@@ -646,6 +653,56 @@ def lt_order_embedding_of_surjective :
 
 end surjective
 
+section bijective
+
+variables (hf : function.bijective f)
+include hf
+
+private lemma le_comap_bot_of_bijective : comap f ⊥ ≤ I :=
```

Comment:
```suggestion
lemma comap_bot_le_of_bijective : comap f ⊥ ≤ I :=
```
If you make the lemma `private` nobody can use it outside this file. In practice, it turns out that there will always be someone who wants to use this 3 months from now :wink: